modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -331,6 +333,54 @@ class InstructionsTest {
         assert(getAccountAmount() == 40U)
     }
 
+    @Test
+    @WithIroha
+    fun `instruction failed`(): Unit = runBlocking {
+        client.sendTransaction {
+            account(ALICE_ACCOUNT_ID)
+            fail("FAIL MESSAGE")
+            buildSigned(ALICE_KEYPAIR)
+        }.also {
+            Assertions.assertThrows(ExecutionException::class.java) {
+                it.get(10, TimeUnit.SECONDS)
+            }
+        }
+    }
+
+    @Test
+    @WithIroha
+    fun `remove asset instruction committed`(): Unit = runBlocking {
+        val assetKey = "key"
+        val assetValue = "value".asValue()
+
+        client.sendTransaction {
+            account(ALICE_ACCOUNT_ID)
+            registerAsset(DEFAULT_ASSET_DEFINITION_ID, AssetValueType.Store())
+            storeAsset(DEFAULT_ASSET_ID, assetKey, assetValue)
+            buildSigned(ALICE_KEYPAIR)
+        }.also {
+            Assertions.assertDoesNotThrow {
+                it.get(10, TimeUnit.SECONDS)
+            }
+        }
+
+        val assetBefore = getDefaultAsset()
+        assertEquals(assetBefore.value.cast<AssetValue.Store>().metadata.map[assetKey], assetValue)
```

Comment:
Please swap position of arguments. First must be expected thing, next must be actual thing. This is important due if assertion fails when output in console will be confusing.